apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatPreviewView.kt

```diff
@@ -83,7 +83,7 @@ fun ChatPreviewView(chat: Chat, chatModelIncognito: Boolean, currentUserProfileD
     val ci = chat.chatItems.lastOrNull()
     if (ci != null) {
       MarkdownText(
-        if (!ci.meta.itemDeleted) ci.text else generalGetString(R.string.marked_deleted_description),
+        if (!ci.meta.itemDeleted) ci.text else if (ci.isPermDeleted) generalGetString(R.string.deleted_description) else generalGetString(R.string.marked_deleted_description),
```

Comment:
CIContent.RcvDeleted items didn't have flag itemDeleted before, it was basically unused after we changed group messages deletion (even earlier this flag was used to hide deleted but pending group messages). So this check is unnecessary and instead defaultDeleted item (in model) should have itemDeleted set to false, then the preview would be showing a different text.
```suggestion
        if (!ci.meta.itemDeleted) ci.text else generalGetString(R.string.marked_deleted_description),
```

Reply:
Thanks for the info